hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/BucketArgs.java

```diff
@@ -228,12 +269,23 @@ public Builder() {
       return this;
     }
 
+    public BucketArgs.Builder setEcReplicationConfig(
+        ECReplicationConfig ecReplConfig) {
+      ecReplicationConfig = ecReplConfig;
+      return this;
+    }
+
 
     /**
      * Constructs the BucketArgs.
      * @return instance of BucketArgs.
      */
     public BucketArgs build() {
+      if(ecReplicationConfig!=null){
+        return new BucketArgs(versioning, storageType, acls, metadata,
+            bucketEncryptionKey, sourceVolume, sourceBucket, quotaInBytes,
+            quotaInNamespace, ecReplicationConfig);
+      }
```

Comment:
If we just add a parameter to the original constructor, we can just pass ecReplicationConfig, and if it is null, it doesn't matter.